### DISCUSSION TITLE
请问英文候选词、预测词如何显示，也没有找到更换词典入口

### DISCUSSION THREAD

**Discussion by Superpengun** [fcitx5-android/fcitx5-android#206] [Q&A | 问答]
使用org.fcitx.fcitx5.android_21版本
![image](https://i.328888.xyz/2023/03/02/FiDj8.png)<br>
![image](https://i.328888.xyz/2023/03/02/FiOft.png)<br>
如图，打开了单词预测功能，但是输入时是直接按键上屏，并没有composing状态及后续预测词

**Comment by rocka** [comment]
有些输入框会告诉输入法“不需要提供拼写检查”，这时候就不会补全，例子有浏览器的地址栏，部分应用或网页的搜索框等。

拼写检查的词典目前不支持更换，用的是 fcitx5 内置的这一份
https://download.fcitx-im.org/data/en_dict-20121020.tar.gz
确实有点年头了 ...

**Comment by dongyuwei** [comment]
如果要更换English输入模式下的词库，是不是修改usr/share/fcitx5/spell/en_dict.fscd 然后重新编译打包？
